eosdis-nasa/earthdata-pub-api
# rename example workflow and create example group and daac

# Description

Creates example group and daac.

## Spec

See Ticket:  [EDPUB-1119](https://bugs.earthdata.nasa.gov/browse/EDPUB-1119)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Create a new user that is part of the example DAAC with full permissions.
4. Create a submission under the example daac and run it to completion
5. confirm there are no errors

---

## Change Log

Creates example group and daac.

* [Add _____](https://github.com/eosdis-nasa/earthdata-pub-api/pull/18/commits/d1624dec446312be2c3951d4faaa8b8b33079ba6)
